desktop/src/main/java/bisq/desktop/main/market/trades/TradesChartsView.java

```diff
@@ -430,11 +437,12 @@ public Number fromString(String string) {
         });
         //noinspection unchecked
         volumeChart.setId("volume-chart");
-        volumeChart.setData(FXCollections.observableArrayList(volumeSeries));
+        //TODO bisqBurnSeries should only be added when model.getCurrencyCode() == "BSQ" ???
```

Comment:
imo no, every time, since BSQ can be used to fund all transaction fees

@chimp1984 ?